src/index.js

```diff
@@ -117,8 +123,8 @@ export const Middleware = {
 
   timestamp: next => msg => next(merge(msg, {
     timestamp: (typeof msg.timestamp === 'Date' ?
-                   (msg.timestamp.getTime() + '000000') :
-                   Date.now() + '000000')
+                   Number((msg.timestamp.getTime() + '000000')) :
```

Comment:
The other option would be to change the To/FromJson methods in Logary itself to represent the timestamps as strings.

Any preference?